src/org/labkey/test/util/ext4cmp/Ext4GridRef.java

```diff
@@ -290,8 +290,8 @@ private WebElement tryStartEditing(int rowIdx, String colName)
                 }
             }
 
-            _test.scrollIntoView(cell); // aligns to bottom
-            _test.scrollBy(0, 100); // bumps up a few rows, above any footers or scrollbars
+            _test.scrollIntoView(cell,true); // aligns to bottom
+//            _test.scrollBy(0, 100); // bumps up a few rows, above any footers or scrollbars
```

Comment:
Remove inaccurate comment and remove `scrollBy` entirely.
```suggestion
            _test.scrollIntoView(cell,true);
```